pkg/services/auth/settings.go

```diff
@@ -0,0 +1,121 @@
+package auth
+
+import (
+	"io/ioutil"
+	"sync"
+
+	"github.com/BurntSushi/toml"
+	"golang.org/x/xerrors"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/util/errutil"
+)
+
+// AuthConfig holds list of connections to LDAP
```

Comment:
Looks like this was copied from `pkg/services/ldap/settings.go`? Unless you have a plan to merge the two, we should remove all these references to `LDAP`.

Reply:
That is correct.  I am removing all references to LDAP.  These are different configuration files.  I do not think we should support a case when OAuth and LDAP configs become merged.